src/group_theory/free_abelian_group.lean

```diff
@@ -345,7 +398,68 @@ instance [comm_monoid α] : comm_ring (free_abelian_group α) :=
       { intros y1 y2 ih1 ih2, rw [mul_add, add_mul, ih1, ih2] } },
     { intros s ih, rw [neg_mul_eq_neg_mul_symm, ih, neg_mul_eq_mul_neg] },
     { intros x1 x2 ih1 ih2, rw [add_mul, mul_add, ih1, ih2] }
-  end
+  end,
   .. free_abelian_group.ring α }
 
+/-- The free abelian group on the empty type is the trivial group. -/
+def pempty_equiv : free_abelian_group pempty ≃+ unit :=
```

Comment:
Can you get this for free by showing `unique (free_abelian_group pempty)` and use the lemma I assume we have for bundled equivs between unique types?

Reply:
Yeah this is shorter. Thanks!